dm/ctl/master/offline_member.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+	"os"
+
+	"github.com/pingcap/dm/dm/ctl/common"
+	"github.com/pingcap/dm/dm/pb"
+
+	"github.com/pingcap/errors"
+	"github.com/spf13/cobra"
+)
+
+// NewOfflineMemberCmd creates an OfflineWorker command
+func NewOfflineMemberCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "offline-member <--master/--worker> <--name master-name/worker-name>",
+		Short: "offline member which has been closed",
+		Run:   offlineMemberFunc,
+	}
+	cmd.Flags().BoolP("master", "m", false, "to offline a master")
+	cmd.Flags().BoolP("worker", "w", false, "to offline a worker")
+	cmd.Flags().StringP("name", "n", "", "specify member name for choosing type")
+	return cmd
+}
+
+func convertOfflineMemberType(cmd *cobra.Command) (string, error) {
+	master, err := cmd.Flags().GetBool("master")
```

Comment:
how about defining const variables for "master" and "worker"?

Reply:
I think we shouldn't use const variables for flags. But the type is using consts now